server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Chapter.kt

```diff
@@ -309,17 +310,52 @@ object Chapter {
                     ")",
             )
 
+        if (!serverConfig.autoDownloadNewChapters.value) {
+            log.debug { "automatic download is not configured" }
+            return
+        }
+
+        // Verify the manga is configured to be downloaded based on it's categories.
+        var mangaCategories = CategoryManga.getMangaCategories(mangaId)
+        // if the manga has no categories, skip checking the categories configuration for automatic
+        // download
+        if (mangaCategories.isNotEmpty()) {
```

Comment:
if a manga has no category it is considered to be in the default category, so you have to check its `includeInDownload` flag

Reply:
Interesting. What's the ideal way to approach that? Should `CategoryManga.getMangaCategories` be updated to return the default category in the list, or should I query for it using `Category.getCategoryById(Category.DEFAULT_CATEGORY_ID)`?

Reply:
You should get the default category manually, CategoryManga.getMangaCategories is used in other places and we don't want to break behaviour